test/ctia/http/generative/es_store_spec.clj

```diff
@@ -56,3 +56,6 @@
 
 (defspec ^:generative api-for-tool-routes-es-store
   prop/api-for-tool-routes)
+
+#_(defspec ^:disabled api-for-scratchpad-routes-es-store
+    prop/api-for-scratchpad-routes)
```

Comment:
You should either remove it before merging or use a `;; TODO: fix it` comment + file an issue not to forget it.